blueocean-web/src/main/js/try.js

```diff
@@ -1,12 +1,7 @@
 var $ = require('jquery-detached').getJQuery();
 
 function getAppUrl() {
-    const rootUrl = $('head').attr('data-rooturl');
-    if (!rootUrl) {
-        return '';
-    } else {
-        return rootUrl;
-    }
+    return rootUrl = $('head').attr('data-rooturl');
```

Comment:
Hmmm ... I'm afraid this is not adding up for me @rpkim. If you're seeing "/jenkins" in the `data-rooturl` attribute in the page source ala ...

![screenshot 2017-01-25 12 10 00](https://cloud.githubusercontent.com/assets/429311/22290200/72030c52-e2f7-11e6-94fa-1d8a6a6d12bb.png)

 ... then Blue Ocean should be opening in "/jenkins/blue" etc. Also, I'm not seeing how the changes here in this PR could possibly fix anything.

Since I'm obviously missing something, I wonder if there's a chance you could record a screen capture showing you browsing to classic jenkins and then trying to open BO via the button. If you can do this, please also show the page contents from the browser dev tools.